### PR TITLE
Fixed JS error on the main page of the site #1013

### PR DESCRIPTION
**Pull request by MSTEWARDSON**
Fixed the undefined error that came up for the 'left' value of viewport on the main page. After looking through the code I determined the issue was caused within the function checkSideBar(). Here the html element 'main-head' was used to call the isOnScreen() function. The issue with this and why the error only happened on the homepage was because the homepage does not use the 'main-head' css class.

To fix this issue there is now a check to see if the 'main-head' element exists using .length. 

Issue: JS error on the main page of the site #1013

### CODE REVIEW

File: assets/js/index.js [hunk 1]
```diff
@@ -68,12 +68,16 @@ $(document).ready(function () {
...
+    if ($(".main-head").length) {
+      if (!$(".main-head").isOnScreen()) {
+        $(".sidebar").addClass("scrolling");
+      } else {
+        $(".sidebar").removeClass("scrolling");
+      }
     }
 
+
+
```
Comment (felops) [inline assets/js/index.js:80]:
```suggestion
```

**Comment by MSTEWARDSON** [comment]
I have removed the extra space in the checkSidebar() function.

**Review by glebkema — COMMENTED** [review]
@MSTEWARDSON Thank you for the quick analysis and fixing the problem!
Could you make the same improvement to the footer code below in this same function?

**Comment by MSTEWARDSON** [comment]
Sure I will work on that now.

**Review by glebkema — APPROVED** [review]
@MSTEWARDSON Super! Thank you very much. We will be glad if you can help us in other tasks.

@shahednasser I propose to combine this PR.

**Comment by shahednasser** [comment]
Awesome thanks for your work and detailed description!

@all-contributors please add @MSTEWARDSON for code and bug

**Comment by MSTEWARDSON** [comment]
Perfect, thanks!

**Pull request merged**